interfaces/ITextToSpeech.h

```diff
@@ -0,0 +1,116 @@
+#ifndef __ITEXTTOSPEECH_H
+#define __ITEXTTOSPEECH_H
+
+#include "Module.h"
+
+namespace WPEFramework {
+namespace Exchange {
+
+    struct EXTERNAL ITextToSpeech : virtual public Core::IUnknown {
+        enum { ID = ID_TEXT_TO_SPEECH };
+
+        struct Configuration {
+        std::string ttsEndPoint;
+        std::string ttsEndPointSecured;
+        std::string language;
+        std::string voice;
+        uint8_t volume;
+        uint8_t rate;
+        };
+
+        enum SpeechState: uint8_t {
+        SPEECH_PENDING = 0,
+        SPEECH_IN_PROGRESS,
+        SPEECH_PAUSED,
+        SPEECH_NOT_FOUND
+        };
+
+        enum TTSErrorDetail: uint8_t {
+        TTS_OK = 0,
+        TTS_FAIL,
+        TTS_NOT_ENABLED,
+        TTS_INVALID_CONFIGURATION,
+        TTS_NO_ACCESS
+        };
+
+
+        struct INotification : virtual public Core::IUnknown {
+            enum { ID = ID_TEXT_TO_SPEECH_NOTIFICATION };
+
+            virtual ~INotification() {}
```

Comment:
As we now require at least std c++ 11, there is now the possibility to use default keyword for constructors and destructors, suggest to use "~INotification() override = default;" in stead of "virtual ~INotification() {}" You do not always see this for all interfaces but this is due to the fact that we started out before std C++ 11 was the default, than the override and default keyword where not always available.